flows/deindex.py

```diff
@@ -283,14 +290,26 @@ async def cleanups_by_s3(
     for i, cleanups_task_batch in enumerate(cleanups_task_batches, start=1):
         logger.info(f"Processing clean-ups task batch #{i}")
 
+        ttl_per_task_s = 30 * 60
```

Comment:
thought: lift this up to the function params with `30*60` as the default value?

Reply:
I'm not sure yet what the best values are, and there already are different values, so I'm going to not be too pre-emptive. We saw this happen with batching values, and we now have different values, and also multiple values ([example](https://github.com/climatepolicyradar/knowledge-graph/blob/fix%2Fproperly-timeout-on-prefect/flows/boundary.py#L54-L55)).